app/views/bets/_display_bets.html.erb

```diff
@@ -17,7 +17,15 @@
             <td class="col-md-2"><%= bet.created_at %></td>
             <td class="col-md-2"><%= bet.expires_at %></td>
             <td class="col-md-2"><%= bet.bet_total %></td>
-            <td class="col-md-4"><%= bet.bet %></td>
+            <td class="col-md-3"><%= bet.bet %></td>
+            <td class="col-md-1">
+              <%=
+                link_to bet, class: 'btn btn-secondary', role: 'button' do
+                  content_tag :i, class: 'fa fa-eye' do
+                  end
```

Comment:
I don't think the `do end` is needed here for your content tag as you aren't sending in anything for the block.

http://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag